CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -275,6 +276,7 @@ function Patient:treatDisease()
   local hospital = self.hospital
 
   hospital:receiveMoneyForTreatment(self)
+  self.pay_amount = 0
```

Comment:
This function is only used for the final treatment. Reset it in `hospital:receiveMoneyForTreatment` instead.

Reply:
This one still needs to be resolved

Reply:
Done